Fix request with get query

btanoue opened the pull request:
fixing contract generation to get method with query
Adding Etag filter to header response generator.
deleting image location header for facebook contract generating.

btanoue committed:
fixing contract generation to get method with query

btanoue committed:
Adding Etag filter to header response generator.

btanoue committed:
deleting image location header for facebook contract generating.

ticolucci commented:
Hello Bruno,
I'm not sure why you had to change the path for GETs with query params.
As far as I remember (and read on lib/pacto/formats/legacy/contract_builder.rb:73), the query params are sent in the `params` field of the contract instead of the `path`.

Also your PR does not include any specs =/
Can you elaborate more please? =)

btanoue committed:
Updating webmock version.
Remove RDStation dependency and resolve a problem when run tests with VCR, cassettes and RestClient.
